EC-84: Revise Cache, Catalog and Checkout sections (#3)
ezsystems/commerce-documentation: docs/guide/catalog/catalog_api/variantproductsnode_and_orderablevariantnode/creating_variantproductnode.md

# Creating VariantProductNode

> Usually you need to create a `VariantProductNode` in your factory,

Reviewer: ```suggestion
Create a `VariantProductNode` in your factory, following the example:
```

> see the following example:

Reviewer: ```suggestion
```
